collectorpb/collector.go

```diff
@@ -1,10 +1,18 @@
 package collectorpb
 
-func (res *ReportResponse) Disable() bool {
-	for _, command := range res.GetCommands() {
+import (
+    "github.com/lightstep/lightstep-tracer-protos/go/lightstep/collector"
+)
+
+type ReportResponseWrapper struct {
+    *collectorpb.ReportResponse
```

Comment:
This wrapper is unnecessary. Can we move this method over to `lightstep-tracer-protos` as well, rather than have this spread out between two repos and then wrapped? `Disable()` is part of the RepostResponse API.

Reply:
I'm confused what you mean when you say "Disable() is part of the RepostResponse API" It isn't a part of the generated code that protoc create, right? I was trying to avoid augmenting a package with generated code in it.